src/javascript/Edit/Edit.actions.js

```diff
@@ -52,7 +52,7 @@ export const registerEditActions = actionsRegistry => {
         menuTarget: 'ContentEditorHeaderActions'
     });
 
-    actionsRegistry.add('action', 'startWorkflow3dots', startWorkflow, {
+    actionsRegistry.add('action', 'startWorkflowHeader', startWorkflow, {
```

Comment:
why updating this name?

Reply:
because it's not a 3 dots menu anymore, the start worklow action is still in the header

Reply:
all right!